docs/ref/django-admin.txt

```diff
@@ -1484,8 +1484,7 @@ override this by passing the desired number of processes, e.g.
 variable.
 
 Django distributes test cases — :class:`unittest.TestCase` subclasses — to
-subprocesses. If there are fewer test cases than configured processes, Django
-will reduce the number of processes accordingly.
```

Comment:
Why you removed: _"Django will reduce the number of processes accordingly"_? This ticket is about clarifying wording `test`/`test case`/`test case class`, etc.

Reply:
Hi @felixxm,as per this
![image](https://github.com/django/django/assets/141929155/730ab3aa-1e86-45a0-95c7-c4a07bca020d) I replaced that line with "If there are fewer test case classes than configured processes."

Reply:
Again, this ticket about clarifying wording `test`/`test case`/`test case class`, etc. The sentence that you removed is correct and should be kept.

Reply:
Also, this is only an example from the ticket description, you should audit all docs and check used wording ( `test`/`test case`/`test case class`).